### ISSUE TITLE
Not possible to ignore custom rules?

### ISSUE THREAD

**Issue by steverice** [cjoudrey/graphql-schema-linter#265]
#240 added a block of code that filters out errors if they are in the ignore list, but only if they are an `instanceof ValidationError`:

```javascript
if (error instanceof ValidationError) {
  …
} else {
  return true;
}
```

Presumably, this is so that plain `GraphQLError`s cannot be accidentally ignored. However, this seems to make it impossible to ignore any custom rules.

The `ValidationError` object that's referenced here is the object exported by `src/validation_error.js` in `graphql-schema-linter`. As `graphql-schema-linter` is expected to be installed globally, other code (where custom rules may live) will be unable to import this reference, at least not without some [ugly workarounds](https://stackfame.com/nodejs-require-global-node-modules).

Theoretically, if the package also containing the custom rules is installed globally, or if `graphql-schema-linter` is being run from the same `node_modules` directory as the custom rule, it would work — but I don't believe this was an intentional restriction.

Am I missing something here? Would it be better to replace the `instanceof` check with something like checking if `ruleName` is defined?

cc @id-ilych

**Comment by cjoudrey** [comment]
Hey @steverice, thanks for opening this issue.

I think this might indeed have been an oversight. Good catch!

Checking for `ruleName` being defined seems better for the use case you mentioned.

Would you mind opening a PR for this?

**Comment by steverice** [comment]
Sure thing. #267 should fix this.
